src/ui/public/paginated_table/paginated_table.html

```diff
@@ -28,7 +28,7 @@
       <tbody kbn-rows="page" kbn-rows-min="paginatedTable.rowsToShow(perPage, page.length)"></tbody>
       <tfoot ng-if="showTotal">
         <tr>
-          <th ng-repeat="col in columns" class="numeric-value">{{col.total | number}}</th>
+          <th ng-repeat="col in columns" class="numeric-value">{{col.total}}</th>
```

Comment:
The only time that col.total is not run through a formatter is for the 'count' function.  So I could just modify the js code to format it there.  Unless you also want to check the result of each formatter as well?